### PR TITLE
[Backend] POST API For Answers under Q&A

### PR DESCRIPTION
**Pull request by arinmodi**
## Issue that this pull request solves 

 Closes: # 705

### Brief description of what is fixed or changed : 
- New Folder Added Called "Q&A"
- Moved Question folder to Q&A
- Create a new folder answer in Q&A
- Create a new model for answer 
- added a new route "\answers"

## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

## Screenshots

image

image

## Other information

Any other information that is important to this pull request

### CODE REVIEW

File: backend/app/routes/Q&A/question/post.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const to = require('await-to-js').default;
-const QUESTION = require('../../models/question');
-const { ErrorHandler } = require('../../../helpers/error');
-const constants = require('../../../constants');
+const QUESTION = require('../../../models/question');
```
Comment (Kajol-Kumari) [inline backend/app/routes/Q&A/question/post.js:2]: Can we please use `question` ie kebab-case in here?

File: backend/app/routes/Q&A/answers/post.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+const to = require('await-to-js').default;
+const ANSWER = require('../../../models/answers');
```
Comment (Kajol-Kumari) [inline backend/app/routes/Q&A/answers/post.js:2]: Please use `answer` here instead

File: backend/app/routes/Q&A/answers/index.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+const router = require('express').Router({ mergeParams: true });
+const postAnswer = require('./post');
+const AnswerValidationSchema = require('./@validationSchema');
```
Comment (Kajol-Kumari) [inline backend/app/routes/Q&A/answers/index.js:3]: Please use `answerValidationSchema` i.e camelCase here.

File: backend/app/routes/Q&A/answers/@validationSchema/index.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+const Joi = require('joi');
+
+const AnswerValidationSchema = Joi.object().keys({
```
Comment (Kajol-Kumari) [inline backend/app/routes/Q&A/answers/@validationSchema/index.js:3]: Please use `answerValidationSchema` i.e camelCase here

File: backend/app/models/answers.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+        type : Number,
+        default : 0
+    },
+    created_on : {
+        type : Date,
+        required : true
+    }
+}, { timestamps: { createdAt: 'createdAt', updatedAt: 'updatedAt' } });
+
+module.exports = mongoose.model('ANSWERS', answerSchema);
```
Comment (Kajol-Kumari) [inline backend/app/models/answers.js:35]: Please use `answers` here instaed

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@arinmodi please do the required minor changes.

**Comment by arinmodi** [comment]
Hey @Kajol-Kumari, I have done the requested changes

**Review by Kajol-Kumari — APPROVED** [review]
LGTM 👍

**Pull request merged**